README.md

```diff
@@ -12,29 +12,40 @@ As an application wiki, XWiki allows for the storing of structured data and the
 
 <!-- generated with pandoc -f gfm --toc -o readme-toc.md README.md -->
 
+- [What is XWiki](#what-is-xwiki)
+- [Table of contents](#table-of-contents)
 - [Introduction](#introduction)
 - [How to use this image](#how-to-use-this-image)
-    -	[Pulling existing image](#pulling-an-existing-image)
-        -	[Using docker run](#using-docker-run)
-        -	[Using docker-compose](#using-docker-compose)
-        -	[Using Docker Swarm](#using-docker-swarm)
-    -	[Using an external Solr service](#using-an-external-solr-service)
-        -	[Preparing Solr container](#preparing-solr-container)
-        -	[Docker run example](#docker-run-example)
-        -	[Docker Compose example](#docker-compose-example)
-    -	[Configuring Tomcat](#configuring-tomcat)
-    -	[Building](#building)
+  - [Pulling an existing image](#pulling-an-existing-image)
+    - [Using docker run](#using-docker-run)
+      - [Starting MySQL](#starting-mysql)
+      - [Starting MariaDB](#starting-mariadb)
+      - [Starting PostgreSQL](#starting-postgresql)
+      - [Starting XWiki](#starting-xwiki)
+    - [Using docker-compose](#using-docker-compose)
+      - [For MySQL on Tomcat](#for-mysql-on-tomcat)
+      - [For MariaDB on Tomcat](#for-mariadb-on-tomcat)
+      - [For PostgreSQL on Tomcat](#for-postgresql-on-tomcat)
+    - [Using Docker Swarm](#using-docker-swarm)
+      - [MySQL Example](#mysql-example)
+      - [PostgreSQL Example](#postgresql-example)
+  - [Using an external Solr service](#using-an-external-solr-service)
+      - [Docker run example](#docker-run-example)
+      - [Docker Compose example](#docker-compose-example)
+  - [Configuring Tomcat](#configuring-tomcat)
+  - [Building](#building)
```

Comment:
Note that the tabs were added voluntarily as otherwise it failed the validation on dockerhub official images.

Now, it's true that since then, we've moved away from putting the doc there and we just link to our doc from https://hub.docker.com/_/xwiki

However, if we want to change this, we need an agreement. Not sure it's the best place to do it in this PR. WDYT?

Reply:
My formatter change it when I'd save the page and rewrite the summary from the title. I will redo the changes.
Nevertheless, I think it could need a refresh to check if it matches the correct title.

I will leave it as before for now